pkg/middleware/request_tracing.go

```diff
@@ -59,17 +62,17 @@ func RequestTracing() web.Handler {
 		// Only call span.Finish when a route operation name have been set,
 		// meaning that not set the span would not be reported.
 		if routeOperation, exists := RouteOperationNameFromContext(c.Req.Context()); exists {
-			defer span.Finish()
-			span.SetOperationName(fmt.Sprintf("HTTP %s %s", req.Method, routeOperation))
+			defer span.End()
+			span.SetName(fmt.Sprintf("HTTP %s %s", req.Method, routeOperation))
 		}
 
 		status := rw.Status()
 
-		ext.HTTPStatusCode.Set(span, uint16(status))
-		ext.HTTPUrl.Set(span, req.RequestURI)
-		ext.HTTPMethod.Set(span, req.Method)
+		span.SetAttributes(attribute.Int("HTTP response status code", status))
+		span.SetAttributes(attribute.String("HTTP request URI", req.RequestURI))
+		span.SetAttributes(attribute.String("HTTP request method", req.Method))
```

Comment:
Think we should keep the old names. Maybe even expose these as constants so that the HTTP Client tracing middleware can use it as well?

```suggestion
		span.SetAttributes(attribute.Int("http.status_code", status))
		span.SetAttributes(attribute.String("http.url", req.RequestURI))
		span.SetAttributes(attribute.String("http.method", req.Method))
```